x-pack/plugins/security_solution/public/cases/containers/api.ts

```diff
@@ -273,20 +271,3 @@ export const getActionLicense = async (signal: AbortSignal): Promise<ActionLicen
   });
   return response;
 };
-export const getFields = async (
```

Comment:
I exposed this call in the flyout I had while I was working on my demo UI, but I am actually not going to expose the fields and instead have a call for the mappings. we have the get fields call in the `caseClient` call, so we do not need it here